<launch>

    <arg name="events_topic_right" default="/prophesee/camera/right/cd_events_buffer"/>
    <arg name="imu_topic_right" default="/prophesee/camera/right/imu"/>
    <arg name="trigger_topic_right" default="/prophesee/camera/right/extTrigger"/>
    
    <arg name="events_topic_left" default="/prophesee/camera/left/cd_events_buffer"/>
    <arg name="imu_topic_left" default="/prophesee/camera/left/imu"/>
    <arg name="trigger_topic_left" default="/prophesee/camera/left/extTrigger"/>
    
    <arg name="event_packet_size" default="100000"/>
    <arg name="imu_packet_size" default="1000"/>
    <arg name="output_root" default=""/>
    <arg name="v" default="0"/>
    

  <!-- Recorder left -->
  <node name="prophesee_recorder_left" pkg="prophesee_recorder" type="prophesee_recorder_node" args="
    --imu_topic=$(arg imu_topic_left)
    --event_topic=$(arg events_topic_left)
    --trigger_topic=$(arg trigger_topic_left)
    --output_root=$(arg output_root)
    --v=$(arg v)
    " output="screen"/>

  <!-- Recorder right -->
  <node name="prophesee_recorder_right" pkg="prophesee_recorder" type="prophesee_recorder_node" args="
    --imu_topic=$(arg imu_topic_right)
    --event_topic=$(arg events_topic_right)
    --trigger_topic=$(arg trigger_topic_right)
    --output_root=$(arg output_root)
    " output="screen"/>

</launch>
